hknweb/academics/models/logistics.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+
+from .base_models import AcademicEntity
+
+
+class Course(AcademicEntity):
+    pass
+
+
+class Department(AcademicEntity):
+    name = models.TextField(max_length=200)
+    abbr = models.TextField(max_length=100)
+
+
+class Instructor(AcademicEntity):
+    instructor_id = models.TextField(max_length=500)

```

Comment:
Do we need to specify primary key for this?

Reply:
The default primary key is just a Django generated `id` field. If your question is "why isn't the `instructor_id` the primary key", it should be, whoops. There might be a bit of additional complexity down the road if we need to change primary keys from say a `first_name`/`last_name` hash to the instructors CalnetID, but this is warranted.